library/src/main/java/io/github/wax911/library/annotation/processor/fragment/FragmentAnalyzer.kt

```diff
@@ -1,9 +1,9 @@
 package io.github.wax911.library.annotation.processor.fragment
 
 /**
- * A contract for something that can parse a query and provide a full analysis of what fragments are referenced, and
- * whether the fragments are defined with the query.
+ * A contract for something that can parse graphql content and provide a full analysis of what fragments are referenced,
+ * and whether the fragments are defined with the query.
  */
 interface FragmentAnalyzer {
-    fun analyzeFragments(query: String): Set<FragmentAnalysis>
+    fun analyzeFragments(graphqlContent: String): Set<FragmentAnalysis>
```

Comment:
The incoming content can be either a query _OR_ a fragment. So, changed this var name in a few places.